ivy/functional/frontends/numpy/sorting_searching_counting/searching.py

```diff
@@ -83,7 +83,5 @@ def nanargmax(a, /, *, axis=None, out=None, keepdims=False):
 
 
 @to_ivy_arrays_and_back
-@from_zero_dim_arrays_to_scalar
```

Comment:
Can you reinstate this function please? It is causing a key error in the test collection which is preventing `test_numpy_extract` from running. Thanks!

Reply:
Done..